Neos.ContentRepository.Dbal/composer.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "neos/contentrepository-dbal",
```

Comment:
so we group now here both the `DbalSchemaDiff` as well as the subscription store implementation. That makes sense because the subscription store needs the diff... it a little weird that now because the `neos/contentgraph-doctrinedbaladapter` depends on the diff and thus brings the subscription engine and the dbal doctrine store.
On the other hand this seems like a good cut. A postgres graph can now still use the common dbal subscription engine without fetching the `doctrinedbaladapter` into the boat ... so 👍